packages/components/addon/components/hds/code-block/index.js

```diff
@@ -107,18 +107,20 @@ export default class HdsCodeBlockIndexComponent extends Component {
     const grammar = Prism.languages[language];
 
     if (code) {
-      if (language && grammar) {
-        this.prismCode = htmlSafe(Prism.highlight(code, grammar, language));
-      } else {
-        this.prismCode = htmlSafe(Prism.util.encode(code));
-      }
-
-      // Force prism-line-numbers plugin initialization, required for Prism.highlight usage
-      // See https://github.com/PrismJS/prism/issues/1234
-      Prism.hooks.run('complete', {
-        code,
-        element,
-      });
+      setTimeout(() => {
+        if (language && grammar) {
+          this.prismCode = htmlSafe(Prism.highlight(code, grammar, language));
+        } else {
+          this.prismCode = htmlSafe(Prism.util.encode(code));
+        }
+
+        // Force prism-line-numbers plugin initialization, required for Prism.highlight usage
+        // See https://github.com/PrismJS/prism/issues/1234
+        Prism.hooks.run('complete', {
+          code,
+          element,
+        });
+      }, 100);
```

Comment:
I tried using `next` to replace this `setTimeout` and it still functions.

I tried using `schedule` instead and got errors but maybe I was using it wrong so I'll experiment a bit more.

Reply:
OK, then let's wait for @alex-ju to come back from PTO and we can then decide how to take it from here.

Reply:
It seems to work although I can do more testing.

Reply:
if we could also test this change to make sure it addresses the actual need in `boundary` (where you identified the issue) that would be even better – let me know if I can be of help

Reply:
@alex-ju I tried figuring out how to add an integration test but haven't been able to figure out a way to do it.

I can try to test it just in Boundary next though.

Reply:
cool! I can look into the integration test bit

Reply:
I tested the CodeBlock updates in Boundary and everything worked as expected. The content updates dynamically.